docs/spec/spec.md

```diff
@@ -143,11 +146,22 @@ Behaviour of the `generate` command,
 bal persist push
 ```
 
```

Comment:
Add a comment here saying `The is not supported.`

Reply:
updated in [c6f8aa2](https://github.com/ballerina-platform/persist-tools/pull/148/commits/c6f8aa200f6cd6b2665e3f091a75d12c1ef59a0f)